src/Datadog.Trace/Vendors/Serilog/Parsing/PropertyToken.cs

```diff
@@ -60,7 +60,8 @@ public PropertyToken(string propertyName, string rawText, string formatObsolete,
         /// <param name="alignment">The alignment applied to the property, if any.</param>
```

Comment:
How did you spot the above `readonly int? _position;` change? Did you have to manually revert that?

Reply:
I checked the commit history on the vendor directly and combed through them.
I manually changed it back.